src/wmrc/config.py

```diff
@@ -35,8 +35,8 @@
 
 COUNTIES_ITEMID = "90431cac2f9f49f4bcf1505419583753"
 
-FACILITIES_LAYER_ITEMID = "d6736e9525274eadb7461b2fd08c426d"  #: Beta version
-# FACILITIES_LAYER_ITEMID = "056bbc52ff3240f6b69666750a61aeff"  #: Live version
+# FACILITIES_LAYER_ITEMID = "d6736e9525274eadb7461b2fd08c426d"  #: Beta version
+FACILITIES_LAYER_ITEMID = "056bbc52ff3240f6b69666750a61aeff"  #: Live version
```

Comment:
Would this be easier to maintain via an environment variable?

Reply:
It might be, but I'd still need it locally for local runs so I'm just going to run with it for now.